pkgs/games/mindustry/default.nix

```diff
@@ -63,15 +91,26 @@ let
   # fake build to pre-download deps into fixed-output derivation
   deps = stdenv.mkDerivation {
     pname = "${pname}-deps";
-    inherit version src postPatch;
+    inherit version unpackPhase;
+    postPatch = cleanupMindustrySrc;
+
     nativeBuildInputs = [ gradle_6 perl ];
     # Here we build both the server and the client so we only have to specify
     # one hash for 'deps'. Deps can be garbage collected after the build,
     # so this is not really an issue.
-    buildPhase = ''
-      ${preBuild}
-      gradle --no-daemon desktop:dist -Pbuildversion=${buildVersion}
-      gradle --no-daemon server:dist -Pbuildversion=${buildVersion}
+    buildPhase = preBuild + ''
+      cat << EOF >> build.gradle
+      task resolveDependencies {
```

Comment:
I wonder if upstream would accept a patch that adds this

Reply:
Good idea, we'll find out: https://github.com/Anuken/Mindustry/pull/4302